.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -98,37 +129,38 @@ def track_prs():
         # SLO, nudge in bold if not.
         message = pr_message(delta, pr_info.html_url, pr_info.title, delta_days, delta_hours)
 
+        if (needs_api_review(labels, repo, pr_info)):
+            add_reminders(pr_info.assignees, api_review_and_prs, message, API_REVIEWERS)
+
         # If the PR has been out-SLO for over a day, inform on-call
         if delta > datetime.timedelta(hours=get_slo_hours() + 36):
             stalled_prs = stalled_prs + message
 
         # Add a reminder to each maintainer-assigner on the PR.
-        has_maintainer_assignee = add_reminders(pr_info.assignees, maintainers_and_prs, message)
+        has_maintainer_assignee = add_reminders(
+            pr_info.assignees, maintainers_and_prs, message, MAINTAINERS)
 
         # If there was no maintainer, track it as unassigned.
         if not has_maintainer_assignee:
-            # don't bother assigning maintainer WIPs.
-            if pr_info.draft and pr_info.user.login in maintainers_and_prs.keys():
-                continue
             maintainers_and_prs['unassigned'] = maintainers_and_prs['unassigned'] + message
 
     # Return the dict of {maintainers : PR notifications}, and stalled PRs
-    return maintainers_and_prs, stalled_prs
+    return maintainers_and_prs, api_review_and_prs, stalled_prs
```

Comment:
might want to update the comment about this with the return val, but non-blocking

Reply:
Good call - will do in a follow-up, as it's late to do today and I'd like to get the blocking bits checked in before tomorrow's run :-)